src/Validation/OrderPropertyValidator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Validation;
+
+/**
+ * @method canHaveMultiCollo(?bool $value): bool
+ * @method canHaveSignature(?bool $value): bool
+ * @method canHaveDate(?string $value): bool
+ */
+trait OrderPropertyValidator
+{
+    public function __call(string $methodName, $args): bool
+    {
+        $schema = $this->getValidationSchema();
+        $keys   = self::METHOD_TO_KEYS[$methodName];
+        $result = $this->walkArrayByKeys($schema, $keys);
```

Comment:
Ik zou hier $args[0] in een variabele laden zodat duidelijk is wat dat is, en als die ontbreekt kan je ook gelijk ophouden met deze method, als je 2 keer een waarde uit een array moet halen, is het in alle gevallen die mij bekend zijn sneller (goedkoper voor php) en makkelijker om die (de eerste keer) in een locale variabele de laden.